rain deploy fails with "error removing temporary template file" on Windows.

My environment is follows.

* Windows 10 (20H2) + PowerShell 7.1.2
* Rain v1.1.1

`rain deploy` command fails with "error removing temporary template file" on Windows.
Template file type didn't matter. Any template file failed.

```powershell
C:\temp> rain --version
Rain v1.1.1 windows/amd64

C:\temp> $env:AWS_PROFILE="my_profile"
C:\temp> rain deploy .\sample.yaml
Preparing template '.\sample.yaml' ˙ ·
Loading AWS config ˙ ·
Preparing template '.\sample.yaml' .·
error removing temporary template file 'C:\Users\<Username>\AppData\Local\Temp\993670691': remove C:\Users\<Username>\AppData\Local\Temp\993670691: The process cannot access the file because it is being used by another process.
```

Debug message is here.

```powershell
C:\temp> $env:AWS_PROFILE="my_profile"
C:\temp> rain deploy .\sample.yaml --debug
Preparing template '.\sample.yaml' ˙ ·
Loading AWS config ˙ ·
Preparing template '.\sample.yaml' ˙ ·
Preparing template '.\sample.yaml' ˙ ·
Preparing template '.\sample.yaml' ·˙.DEBUG: Artifact bucket: rain-artifacts-<my account id>-ap-northeast-1

Preparing template '.\sample.yaml'  .˙DEBUG: Package output:
Successfully packaged artifacts and wrote output template to file C:\Users\<Username>\AppData\Local\Temp\720697167.
Execute the following command to deploy the packaged template
aws cloudformation deploy --template-file C:\Users\<Username>\AppData\Local\Temp\720697167 --stack-name <YOUR STACK NAME>

DEBUG: Loading packaged template file
DEBUG: Removing temporary template file: C:\Users\<Username>\AppData\Local\Temp\720697167

panic: error removing temporary template file 'C:\Users\<Username>\AppData\Local\Temp\720697167': remove C:\Users\<Username>\AppData\Local\Temp\720697167: The process cannot access the file because it is being used by another process. [recovered]
        panic: error removing temporary template file 'C:\Users\<Username>\AppData\Local\Temp\720697167': remove C:\Users\<Username>\AppData\Local\Temp\720697167: The process cannot access the file because it is being used by another process.

goroutine 1 [running]:
github.com/aws-cloudformation/rain/internal/cmd.execute.func1(0xc0000abf58)
        /__w/rain/rain/internal/cmd/wrap.go:77 +0x23b
panic(0x1ac1140, 0xc000586360)
        /usr/local/go/src/runtime/panic.go:969 +0x1c7
github.com/aws-cloudformation/rain/internal/cmd/deploy.packageTemplate.func1(0xc0000061c0)
        /__w/rain/rain/internal/cmd/deploy/util.go:170 +0x198
github.com/aws-cloudformation/rain/internal/cmd/deploy.packageTemplate(0xc00003a0d0, 0xd, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /__w/rain/rain/internal/cmd/deploy/util.go:192 +0x40c
github.com/aws-cloudformation/rain/internal/cmd/deploy.glob..func1(0x27ad820, 0xc000005760, 0x1, 0x2)
        /__w/rain/rain/internal/cmd/deploy/deploy.go:67 +0x255
github.com/spf13/cobra.(*Command).execute(0x27ad820, 0xc000005740, 0x2, 0x2, 0x27ad820, 0xc000005740)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:854 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x27aea80, 0x27af260, 0xc0000abf48, 0x698178)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/aws-cloudformation/rain/internal/cmd.execute(0x27aea80, 0x0)
        /__w/rain/rain/internal/cmd/wrap.go:86 +0x67
github.com/aws-cloudformation/rain/internal/cmd.Execute(0x27aea80)
        /__w/rain/rain/internal/cmd/wrap.go:95 +0x32
main.main()
        /__w/rain/rain/cmd/rain/main.go:23 +0x34
C:\temp>
```

It seems like file handle need to explicitly be closed at [deploy.packageTemplate](https://github.com/aws-cloudformation/rain/blob/f812b854ba75d08b931862fc461b6fc9dc25b79b/internal/cmd/deploy/util.go#L161-L193) function.

https://github.com/aws-cloudformation/rain/blob/f812b854ba75d08b931862fc461b6fc9dc25b79b/internal/cmd/deploy/util.go#L161-L193

--- a/internal/cmd/deploy/util.go
+++ b/internal/cmd/deploy/util.go
@@ -165,6 +165,7 @@ func packageTemplate(fn string) cft.Template {
 	}
 	defer func() {
 		config.Debugf("Removing temporary template file: %s", outputFn.Name())
+		outputFn.Close()
 		err := os.Remove(outputFn.Name())
 		if err != nil {
 			panic(ui.Errorf(err, "error removing temporary template file '%s'", outputFn.Name()))

